stories/core/SideNavigation.stories.tsx

```diff
@@ -82,3 +94,96 @@ export const ActiveItem: Story<SideNavigationProps> = (args) => {
     />
   );
 };
+
+export const Submenu: Story<SideNavigationProps> = (args) => {
+  const itemLabels = ['Home', 'Issues', 'Documents'];
```

Comment:
Maybe have an object with labels and icons combined.

Reply:
Done